src/pages/community-champions.js

```diff
@@ -193,6 +211,82 @@ const LeadByExampleTitle = styled(H4)`
     }
 `;
 
+const BenefitsAndRewardsGrid = styled('div')`
+    display: grid;
+    grid-row-gap: ${BENEFITS_AND_REWARDS_GRID_ROW_GAP};
+    @media ${screenSize.mediumAndUp} {
+        grid-column-gap: ${size.mediumLarge};
+        grid-template-columns: repeat(2, 1fr);
+    }
+    @media ${screenSize.largeAndUp} {
+        grid-template-columns: repeat(4, 1fr);
+    }
+`;
+
+const BenefitsAndRewardsItemContainer = styled('div')`
+    display: flex;
+    flex-direction: column;
+`;
+
+const BenefitsAndRewardsSectionTitle = styled(H5)`
+    margin-bottom: ${size.default};
+    margin-top: ${size.mediumLarge};
+    text-align: center;
+    @media ${screenSize.upToMedium} {
+        font-size: ${fontSize.medium};
+        line-height: ${lineHeight.medium};
+    }
+    @media ${screenSize.upToLarge} {
+        margin-bottom: ${size.xsmall};
+    }
+`;
+
+const BenefitsAndRewardsImageContainer = styled('div')`
+    display: flex;
+    height: ${BENEFITS_AND_REWARDS_IMAGE_CONTAINER_HEIGHT};
+    justify-content: center;
+    max-width: 100%;
+`;
+
+const BenefitsAndRewardsContainer = styled('div')`
+    background-color: ${({ theme }) => theme.colorMap.devBlack};
+    margin: 0 -${size.xxlarge};
+    padding: ${size.xlarge} ${size.xxlarge};
+    @media ${screenSize.upToLarge} {
+        margin: 0 -${size.default};
+        padding: ${size.large} ${size.default};
+    }
+`;
+
+const BulletText = styled(P2)`
+    color: ${({ theme }) => theme.colorMap.devWhite};
+    margin-bottom: 0;
+    @media ${screenSize.upToMedium} {
+        font-size: ${fontSize.small};
+        line-height: ${lineHeight.small};
+    }
+`;
+
+const GreenBulletedListWithNoMargin = styled(GreenBulletedList)`
+    margin: 0;
+    li {
+        margin-bottom: ${size.xsmall};
+    }
+`;
+
+const BenefitsAndRewardsItem = ({ icon, title, bullets }) => (
```

Comment:
Nice reuse! Should we alphabetize the props?

Reply:
yes :p